setup.py

```diff
@@ -87,19 +88,28 @@
     # "keras2onnx @ git+git://github.com/onnx/keras-onnx.git@cbdc75cb950b16db7f0a67be96a278f8d2953b48#egg=keras2onnx",
 ]
 extras["torch"] = ["torch>=1.0"]
+
 extras["flax"] = ["jaxlib==0.1.55", "jax>=0.2.0", "flax==0.2.2"]
+if os.name == "nt":  # windows
+    extras["flax"] = [] # jax is not supported on windows
```

Comment:
Can we define `extras["retrieval"]` before this line with just `"datasets"`  and append "faiss-cpu" here (it only works on Windows too).

Reply:
I think it only works on non-windows (linux + osx), but good comment nonetheless!

Reply:
Yes non-Windows, sorry ;-)